src/main/java/com/bookshop/controller/BookController.java

```diff
@@ -28,30 +32,36 @@ public class BookController {
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books",
+            description = "Get list of books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
```

Comment:
you may make use of @PageableDefault here in case client did not provide pagination parameters

Reply:
agree with @teract10s 

`@PageableDefault` would be beneficial here